### PR TITLE
Remove references to Timer API as beta

### PR DESCRIPTION
**Pull request by rudyardrichter**

### CODE REVIEW

File: pypi/README.md [hunk 1]
```diff
@@ -19,18 +19,14 @@ transfer to and from as well as other options relevant to the transfer.
...
-
-Once the appropriate version of Python is ready, install with `pip
-install globus-timer-cli`. This will create a new command line
-utility, `globus-timer` which can be used for all interactions with
-the service. Online documentation is always available using the
-`--help` option on any command. So, `globus-timer --help` will provide
-information about the options on the command while `globus-timer job
---help` will provide help text specific to working with jobs. When in
-doubt, add `--help` to a command for guidance on next steps.
+This CLI requires Python version 3.5 or higher. Once the appropriate version of
```
Comment (rudyardrichter) [inline pypi/README.md]: Also here remove the broken link to the main docs (that page no longer exists).

Comment (kurtmckee) [inline pypi/README.md]: Python 3.6 or higher is required per `pyproject.toml`.

```suggestion
This CLI requires Python version 3.6 or higher. Once the appropriate version of
```

File: pypi/README.md [hunk 2]
```diff
@@ -19,18 +19,14 @@ transfer to and from as well as other options relevant to the transfer.
...
+Python is ready, install with `pip install globus-timer-cli`. This will create
+a new command line utility, `globus-timer` which can be used for all
```
Comment (kurtmckee) [inline pypi/README.md]:
```suggestion
a new command line utility, `globus-timer`, which can be used for all
```

**Review by kurtmckee — CHANGES_REQUESTED** [review]

**Comment by rudyardrichter** [comment]
Thanks @kurtmckee, changes applied.

**Review by kurtmckee — APPROVED** [review]

**Pull request merged**